index.js

```diff
@@ -62,8 +62,6 @@ module.exports = {
         disableDecoratorTransforms: false,
         enableTypeScriptTransform: true,
 
-        throwUnlessParallelizable: true,
```

Comment:
any reason we didn't leave this? if we aren't parallelizable something bad has gone wrong. I've worked pretty hard to keep things parallelizable around the ecosystem opening fixes when hitting issues.

Reply:
I wasn't able to trace the error back to the root cause. I can give it another look, but I'm going to consider this non-blocking and decouple the investigation. Frankly I considered this option as being old enough that I wasn't sure it was still something meaningful.

A conversion of this whole project to a v2 addon would simplify the dependencies and I presume make the build parallelization concerns moot (at least for consumers of this library), and I suspect that is the meta.